mod_reforged/hooks/skills/actives/throw_smoke_bomb_skill.nut

```diff
@@ -0,0 +1,31 @@
+::mods_hookExactClass("skills/actives/throw_smoke_bomb_skill", function(o) {
+	local getTooltip = o.getTooltip;
+	o.getTooltip = function()
+	{
+		local ret = getTooltip();
+        foreach (index, entry in ret)
+        {
+            if (entry.id != 5) continue;
+			if (entry.text.find("Increases Ranged Defense by ") == null) continue;	// Vanilla has two entries with id 5. That's why we check for a phrase aswell
```

Comment:
Vanilla has the text `Ranged Defense` not `Increases Ranged Defense by`, so you should check for the former.